arch/x86/apic.c

```diff
@@ -197,6 +199,8 @@ void init_apic_timer(void) {
         min_ticks = min(min_ticks, elapsed_ticks);
     }
 
+    cli();
```

Comment:
This should restore the old state, not unconditionally disable interrupts?

Reply:
Yes, will change.